flake.nix

```diff
@@ -8,16 +8,19 @@
     # Custom packages
     polykey-cli.url = "github:MatrixAI/Polykey-CLI";
     polykey-cli.inputs.nixpkgs.follows = "nixpkgs"; # Inheriting the nixpkgs input above; pinned
+
+    gitignore-nix.url = "github:hercules-ci/gitignore.nix";
+    gitignore-nix.inputs.nixpkgs.follows = "nixpkgs";
   };
 
-  outputs = inputs@{ nixpkgs, flake-utils, ... }:
+  outputs = inputs@{ nixpkgs, flake-utils, gitignore-nix, ... }:
```

Comment:
Why do you need to do this?

Reply:
this was apparently an artifact of having the tests and other utilities. now that we have removed it, this is no longer needed. i have updated this in the latest commit.